apps/language_models/src/pipelines/falcon_pipeline.py

```diff
@@ -204,11 +236,10 @@ def compile_falcon(self):
             self.falcon_vmfb_path.parent.absolute(),
             self.falcon_vmfb_path.stem,
             extra_args=[
-                "--iree-hal-dump-executable-sources-to=ies",
+                f"--iree-hal-dump-executable-sources-to=hal_sources_{self.model_name}",
```

Comment:
I've seen this in a few more scripts like [MiniGPT4](https://github.com/nod-ai/SHARK/blob/4fec03a6ccca77d53c2ee3d6da3453c19b5056c8/apps/language_models/src/pipelines/minigpt4_pipeline.py#L301) and I feel we should get rid of this flag OR perhaps add it based on a `--debug` flag.